test/services/LookupServiceSpec.scala

```diff
@@ -96,13 +100,33 @@ class LookupServiceSpec extends ServiceBaseSpec with MockAppConfig {
       }
     }
 
+    "the mongo lookup repository is switched off" should {
+      "call IFS and return the mtdId" in new Test {
+
+        val serviceResponse = Right(reference)
+
+        MockedAppConfig.featureSwitches
+          .returns(Configuration("ifs.enabled" -> true, "mongo-lookup.enabled" -> false))
+          .anyNumberOfTimes()
+
+        MockedLookupRepository.getMtdReference(nino).never()
+        mockGetMtdIdFromIfs(nino).returns(Future.successful(Right(ResponseWrapper(correlationId, ifsReference))))
+        MockedLookupRepository.save(cached).never()
+        MockTimeProvider.now().returns(fixedInstant)
```

Comment:
Can you please add `.never()` so it highlights that `TimeProvider` is not used when the `mongo lookup repository is switched off`, thanks.